trestle/core/commands/assemble.py

```diff
@@ -181,7 +181,8 @@ def assemble_model(cls, model_alias: str, object_type: Type[TLO], args: argparse
 
         # distributed load
         _, _, assembled_model = load_distributed(root_model_filepath)
-        assembled_model_filepath = trestle_root / const.TRESTLE_DIST_DIR / f'{model_alias}.{args.extension}'
+        assembled_model_dir = trestle_root / const.TRESTLE_DIST_DIR / f'{model_alias}s'
```

Comment:
This will not work for the models which already have `s` at the end of the string - you need to have a switch to deal with this.